submissions/zhenyakornilov/js-dom/js/utils.js

```diff
@@ -0,0 +1,12 @@
+function changeTheme() {
+  
+  const checkbox = document.getElementById("switcher");
+  checkbox.addEventListener("change", () => {
```

Comment:
better to separate the handler function from a listener. 
el.addEventListener("change", handler);
function handler {}